spyderlib/restart_app.py

```diff
@@ -63,6 +65,47 @@ def is_pid_running(pid):
         return _is_pid_running_on_unix(pid)
 
 
+def launch_error_message(type_, error=None):
+    """Launch a message box with a predefined error message.
+
+    Parameters
+    ----------
+    type_ : int [CLOSE_ERROR, RESET_ERROR, RESTART_ERROR]
+    """
+    from spyderlib.baseconfig import _
+    from spyderlib.qt.QtGui import QMessageBox, QDialog
+    from spyderlib.utils import icon_manager as ima
+    from spyderlib.utils.qthelpers import qapplication
+
+    messages = {CLOSE_ERROR: _("The previous instance of Spyder has not "
+                               "closed.\nRestart aborted."),
+                RESET_ERROR: _("Spyder could not reset to factory defaults.\n"
+                               "Restart aborted."),
+                RESTART_ERROR: _("Spyder could not restart.\n"
+                                 "Restart aborted.")}

```

Comment:
Another suggestion

    It was not possible to restart Spyder.
    Operation aborted.